fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/DefaultFilter.java

```diff
@@ -59,7 +60,8 @@
     @Override
     public boolean test(final Event event) {
         try {
-            return getMixinTypes(event).anyMatch(fedoraMixins::contains);
+            // NODE_MOVED events do not contain a mixin type
+            return getMixinTypes(event).anyMatch(fedoraMixins::contains) || event.getType() == NODE_MOVED;
```

Comment:
I this going to catch nonFedora stuff moving around in the JCR for MODE system purposes?